website/docs/docs/core/connect-data-platform/cratedb-setup.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "CrateDB setup"
+description: "Read this guide to learn about the CrateDB warehouse setup in dbt."
+id: "cratedb-setup"
+meta:
+  maintained_by: Crate.io, Inc.
+  authors: 'CrateDB maintainers'
+  github_repo: 'crate/dbt-cratedb2'
+  pypi_package: 'dbt-cratedb2'
+  min_core_version: 'v1.0.0'
+  cloud_support: Not Supported
+  min_supported_version: 'n/a'
+  slack_channel_name: 'Community Forum'
+  slack_channel_link: 'https://community.cratedb.com/'
+  platform_name: 'CrateDB'
+  config_page: '/reference/resource-configs/postgres-configs'
```

Comment:
_Let me reiterate this here, to have a threaded conversation about this topic._

@amychen1776 asked:

> Are there any configurations that need to be mentioned in terms of using the adapter like we do for postgres? https://docs.getdbt.com/reference/resource-configs/postgres-configs

Reply:
This is right on the spot, @amychen1776. Thanks for bringing it up.

I've sloppily referred to the PostgreSQL configuration settings here, because I thought to have picked up this strategy from another page that describes a database that is compatible with PostgreSQL. However, I might be wrong, as I can't find it back, and it is a bad idea anyway.

Actually, we need to verify first what CrateDB will be capable of, because we needed to deactivate a few test cases compared to PostgreSQL, where CrateDB lacks relevant features.

- https://github.com/crate/dbt-cratedb2/pull/2

Reply:
In this spirit, we should review our implementation, and then possibly add a dedicated cratedb-configs page, for example like [yellowbrick-configs](https://docs.getdbt.com/reference/resource-configs/yellowbrick-configs) is doing it, that educates readers about the fact it is only supporting a subset of the materialization strategies compared to what the PostgreSQL adapter provides per [postgres-configs](https://docs.getdbt.com/reference/resource-configs/postgres-configs).

Another option, possibly for immediate application, would be to do it like the [risingwave-setup](https://docs.getdbt.com/docs/core/connect-data-platform/risingwave-setup) page is doing it, that is referring to the [no-configs](https://docs.getdbt.com/reference/resource-configs/no-configs) page, that says there are  "no specific configurations for this adapter". This way, we could contribute a relevant page later, but sure enough it would be sweet to get it right and complete from the very beginning.

Can you possibly support here, @hlcianfagna?

Reply:
For pragmatic reasons, mostly to unlock the patch to be merged, I've adjusted it to refer to the [no-configs](https://docs.getdbt.com/reference/resource-configs/no-configs) page now, and created a downstream ticket to track that gap on our realms.

- https://github.com/crate/dbt-cratedb2/issues/7

067d8b819e, included here, slightly improves the no-configs page on your realms. If you don't like it, let me know if I should remove it again. Thanks!

Reply:
I second the idea of going with the no-configs page for the time being. I added a couple of thoughts to [the follow up ticket](https://github.com/crate/dbt-cratedb2/issues/7).

Reply:
Thank you. Resolving this.

Reply:
Thanks!